nodes/widgets/locales/en-US/ui_text.html

```diff
@@ -13,6 +13,16 @@
             <li><b>Text Color:</b> Color picker to define the color to display the text in.</li>
         </ul>
     </p>
+    <h3>HTML Injection</h3>
```

Comment:
```suggestion
    <h3>HTML Content</h3>
```

injection has connotations - maybe say "content" instead?